opencti-platform/opencti-front/src/utils/hooks/useForceUpdate.ts

```diff
@@ -0,0 +1,11 @@
+import { useCallback, useState } from 'react';
+
+const useForceUpdate = () => {
+  const [, setValue] = useState<object>({});
+
+  return useCallback((): void => {
+    setValue({});
+  }, []);
+};
+
+export default useForceUpdate;
```

Comment:
Nice !

Reply:
Personally dont get when we need to use this hook. Could you explain?

Reply:
Actually I don't need it anymore.
We could have some case when we could need this but for now it is unused.

One case is : I use ref to store data because I don't want to be rendered everytime I update it, but some updates on this ref should render => useForceUpdate().

For now I'll just remove it.